### DISCUSSION TITLE
Split from issue 729 - about searx.be location

### DISCUSSION THREAD

**Discussion by unixfox** [searx/searx#2450] [Public Searx instances: feedback]
@ltGuillaume I'm splitting the discussion here because I don't really want to have off-topic. Don't take it personally, but there are already too many comments in the issue #729, so I want to avoid adding more that are off-topic.

----------------------

@ltGuillaume wrote:

> > Just a self-promotion, I'm running a public instance which is designed to work (most of the time) with Google if you are interested: [searx.be](https://searx.be).
> 
> @unixfox So according to [searx.space](https://searx.space) you're running this on a US server? worried

@unixfox wrote:

> > Just a self-promotion, I'm running a public instance which is designed to work (most of the time) with Google if you are interested: [searx.be](https://searx.be).
> 
> @unixfox So according to [searx.space](https://searx.space) you're running this on a US server? worried

> > > Just a self-promotion, I'm running a public instance which is designed to work (most of the time) with Google if you are interested: [searx.be](https://searx.be).
> > 
> > 
> > @unixfox So according to [searx.space](https://searx.space) you're running this on a US server? worried
> 
> No I'm not, it's just the company that is US based, but it's actually hosted in the Netherlands.

@ltGuillaume wrote:

> But if the company is US-based, US laws apply to your instance? (I wonder if that entails much when it comes to a searx instance, but it does feel less trustworthy).

**Comment by unixfox** [comment]
@ltGuillaume

> But if the company is US-based, US laws apply to your instance? (I wonder if that entails much when it comes to a searx instance, but it does feel less trustworthy).

I'm not a lawyer but for hosting, the laws are those from the country where the server is hosted.

My searx instance is in fact hosted in multiple locations, the Netherlands is just a proxy that terminate the TLS. I've multiple processes of Searx hosted in Germany and Paris too and under a different hosting provider.

**Comment by ltguillaume** [comment]
This is great, thx for splitting this from the issue 🙂 I already felt a bit "uncomfortable" to ask in the other discussion, to be honest 😅

So according searx.space the company is Oracle, or at least the servers are run by Oracle?

**Reply by unixfox** [reply]
> This is great, thx for splitting this from the issue slightly_smiling_face I already felt a bit "uncomfortable" to ask in the other discussion, to be honest sweat_smile

Feel free to create a new GitHub discussions and ping the user for the next time :). That's why I spent a lot of time categorizing the Github discussions of Searx.

> So according searx.space the company is Oracle, or at least the servers are run by Oracle?

Yes the main webserver is hosted at Oracle with their free Linux server offer: https://www.oracle.com/cloud/free/. But for the Searx processes those are hosted at Oracle, Azure and Alibaba cloud.

You may wonder why those companies, well in fact I use only their free offers because I'm still a student and I don't have a lot of money, but I'm still interested to help the Searx community so that's why I host it in providers that don't cost me at all.

Once I'll start to earn some money, hopefully this year once I graduate, I'll move my servers to companies that aren't US based like hetzner.

I don't force you to use my instance, if you don't feel it trustworthy that's fine, but I still wanted to explain the reason of using a US based hosting provider.

**Reply by ltguillaume** [reply]
That makes perfect sense. As I am currently using Disroot's instance, which imho is useless due to constant Google captcha errors, I was curious about whether all the mitigations you have applied actually do work. Thanks for elaborating on the way you've deployed your instance.
